nevergrad/functions/perfcap3d/core.py

```diff
@@ -0,0 +1,436 @@
+import json
+import os
+import pathlib
+import typing as tp
+import logging
+import itertools
+import copy
+from enum import Enum
+from uuid import uuid4 as uuid_gen
+from nevergrad.parametrization import parameter as p
```

Comment:
throughout the code we now preferably use:
`import nevergrad as ng`
then `ng.p` when needed

Reply:
Done